### PR TITLE
Clarify 12.1.1 as per #1384

### PR DESCRIPTION
**Pull request by tghosth**
This Pull Request relates to issue #1384

### CODE REVIEW

File: 5.0/en/0x20-V12-Files-Resources.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Although zip bombs are eminently testable using penetration testing techniques,
 
 | # | Description | L1 | L2 | L3 | CWE |
 | :---: | :--- | :---: | :---: | :---: | :---: |
-| **12.1.1** | Verify that the application will not accept large files that could fill up storage or cause a denial of service. | ✓ | ✓ | ✓ | 400 |
+| **12.1.1** | Verify that the application will not accept large files that could cause a denial of service when they are processed. | ✓ | ✓ | ✓ | 400 |
```
Comment (elarlang) [inline 5.0/en/0x20-V12-Files-Resources.md:16]: khm. MODIFIED

**Review by elarlang — CHANGES_REQUESTED** [review]

**Comment by elarlang** [comment]
https://github.com/OWASP/ASVS/issues/1384#issuecomment-1372609416

File: 5.0/en/0x20-V12-Files-Resources.md (hunk 1 above)
Comment (tghosth) [reply 5.0/en/0x20-V12-Files-Resources.md]: Fixed

**Comment by tghosth** [comment]
Ok I modified as per https://github.com/OWASP/ASVS/issues/1384#issuecomment-1374934243

**Review by elarlang — APPROVED** [review]

**Pull request merged**